tests/phpunit/api/v3/RelationshipTest.php

```diff
@@ -322,7 +322,7 @@ public function testRelationshipCreate($version) {
       'relationship_type_id' => $this->_relTypeID,
       'start_date' => '2010-10-30',
       'end_date' => '2010-12-30',
-      'is_active' => 1,
```

Comment:
Can you explain all these changes - I suspect these tests are no longer testing what they were supposed to - perhaps the dates need editing to make the tests still work?

Reply:
I have modified the parameters instead of changing the asserts now wherever possible and reverted back the assertion changes